src/factors/Range2D.jl

```diff
@@ -17,11 +17,14 @@ function (cfo::CalcFactor{<:Point2Point2Range})(rho, theta, xi, lm)
   #
   XX = lm[1] - (rho[1]*cos(theta[1]) + xi[1])
   YY = lm[2] - (rho[1]*sin(theta[1]) + xi[2])
-  #TODO JT - Should this have a sqrt for parametric?
-  # return XX^2 + YY^2 
   return sqrt(XX^2 + YY^2)
 end
 
+#Parametric only function fro Point2Point2Range 
+function (cfo::CalcFactor{<:Point2Point2Range})(rho, xi, lm)
+  # Basically `EuclidDistance`
+  return [rho[1] - norm(lm .- xi)]
+end
```

Comment:
right, understand now.  Best solution would be to drop the additional `theta` variable, but we should look at how best to do together with the Manifolds.jl upgrade -- lots of overlap.

Reply:
I never liked the additional `theta` thing.  Perhaps the solution is the same as JuliaRobotics/IncrementalInference.jl#1051